pygmt/helpers/utils.py

```diff
@@ -220,3 +220,29 @@ def launch_external_viewer(fname):
         subprocess.run(["open", fname], check=False, **run_args)
     else:
         webbrowser.open_new_tab("file://{}".format(fname))
+
+
+def args_in_kwargs(args, kwargs):
+    """
+    Take a list and a dictionary, and determine if any entries in the list are
+    keys in the dictionary.
+
+    This function is used to determine if one of the required arguments is
+    passed to raise a GMTInvalidInput Error.
+
+    Parameters
+    ----------
+    args : list
+        List of required arguments, using the GMT short aliases.
+
+    kwargs : dict
+        The dictionary of kwargs is the  format returned by the _preprocess
+        function in BasePlotting in base_plotting.py. The keys are the GMT
+        short aliases of the parameters.
+
+   Returns
+   --------
+   bool 
+       If one of the required arguments is in `kwargs`.
+    """
```

Comment:
It seems the indentation is incorrect. It's difficult to correct it correct in using the github web editor.

Reply:
@seisman I'll work on this when I get the chance; I'm currently away from home for work, so I don't have access to my main computer.

Reply:
@willschlitzer I just made some changes to this branch, now it should be almost good to merge.